import psycopg2
from psycopg2 import Error

try:
    connection = psycopg2.connect(user="postgres",
                                  password="1488",
                                  host="127.0.0.1",
                                  port="5432",
                                  database="postgres")

    cursor = connection.cursor()
    print("Информация о сервере PostgreSQL")
    print(connection.get_dsn_parameters(), "\n")
    cursor.execute("INSERT INTO product (id, name, price, quantity) VALUES (4, 'Cacao', '30', '1000') ")
    cursor.execute("DELETE FROM product WHERE name ='Coffee' ")
    connection.commit()
    cursor.execute("select * from product where id = '1' ")
    row = cursor.fetchone()
    print(row)

except (Exception, Error) as error:
    print("Ошибка при работе с PostgreSQL", error)
finally:
    if connection:
        cursor.close()
        connection.close()
        print("Соединение с PostgreSQL закрыто")
